src/components/HURUmap/Panel/MobileTabPanel/PanelItem.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from "prop-types";
```

Comment:
And since we only have one item in mobile, I don't think we need to call this `PanelItem`